projects/common-form-elements/src/lib/dynamic-dropdown/dynamic-dropdown.component.ts

```diff
@@ -94,6 +95,7 @@ export class DynamicDropdownComponent implements OnInit, OnChanges, OnDestroy {
 
 
     if (!_.isEmpty(this.depends)) {
+    this.handleDependentFieldChanges();
      this.contextValueChangesSubscription =  merge(..._.map(this.depends, depend => depend.valueChanges)).pipe(
```

Comment:
@bharatkashyap please remove this code

Reply:
@pallakartheekreddy - Missed that. Fixed now